packages/oas/openapi-typescript-codegen/src/openApi/v3/parser/getModelsExpandedRelations.ts

```diff
@@ -59,12 +64,20 @@ const walkNestedRelations = (
   nestedRelation: NestedRelation,
   parentNestedRelation?: NestedRelation
 ) => {
-  const prop =
-    model.export === "all-of"
-      ? findPropInAllOf(nestedRelation.field, model, allModels)
-      : getPropertyByName(nestedRelation.field, model)
+  const prop = ["all-of", "any-of", "all-of"].includes(model.export)
+    ? findPropInAllOf(nestedRelation.field, model, allModels)
+    : getPropertyByName(nestedRelation.field, model)
   if (!prop) {
-    return
+    throw new Error(`x-expanded-relations - field not found
+      Schema: ${rootModel.name}
+      NestedRelation: ${JSON.stringify(nestedRelation, null, 2)}
+      Model: ${JSON.stringify(model.spec, null, 2)}`)
+  }
+  if (["all-of", "any-of", "all-of"].includes(prop.export)) {
```

Comment:
todo: `all-of` duplicated

Reply:
Good catch 🎣. Fixed in 1759dbff8d96ac94586c78697804bea6ff865558